components/ResultsPage/Estimation.tsx

```diff
@@ -0,0 +1,308 @@
+import { useRouter } from 'next/router'
+import { getTranslations, numberToStringCurrency } from '../../i18n/api'
+import { WebTranslations } from '../../i18n/web'
+import { Language, ResultKey } from '../../utils/api/definitions/enums'
+import {
+  BenefitResult,
+  BenefitResultsObject,
+} from '../../utils/api/definitions/types'
+import { useTranslation } from '../Hooks'
+import { EstimatedTotalItem } from './EstimatedTotalItem'
+
+export const Estimation: React.VFC<{
+  partner
+  resultObject
+  resultArray
+  age
+  maritalStatus
+}> = ({ partner, resultObject, resultArray, age, maritalStatus }) => {
+  const tsln = useTranslation<WebTranslations>()
+  const apiTrans = getTranslations(tsln._language)
+  age = Math.round(age)
```

Comment:
passed in props should be immutable. Instead of reassigning, should assign to a local var, "roundedAge" maybe?